go.mod

```diff
@@ -285,8 +285,8 @@ require (
 replace (
 	// branch: v0.27.0.rc3-osmo, current tag: v0.27.0.rc3-osmo
 	github.com/CosmWasm/wasmd => github.com/osmosis-labs/wasmd v0.27.0-rc2.0.20220517191021-59051aa18d58
-	// Our cosmos-sdk branch is:  https://github.com/osmosis-labs/cosmos-sdk, current tag: v0.45.0x-osmo-v11-alpha.2, current branch: osmosis-main
-	github.com/cosmos/cosmos-sdk => github.com/osmosis-labs/cosmos-sdk v0.45.1-0.20220721082708-0aca1df58871
+	// Our cosmos-sdk branch is:  https://github.com/osmosis-labs/cosmos-sdk, current tag: 0.45.0x-osmo-v11-alpha.5 current branch: osmosis-main
```

Comment:
Should we start tagging sdk branch as v11 proper now that we have our v11 cut? If not, should we be calling this v12-alpha?

Reply:
This is made against `osmosis-main` so, in my opinion, this shouldn't be named `v11` in general since main tends to be independent of versioning. Next time we update this, I think we should rename it to something like `v0.45.0x-osmo-main.alpha.1`. Wdyt?